src/http-api/routes/debug.js

```diff
@@ -0,0 +1,24 @@
+const register = require('prom-client').register
+const client = require('prom-client')
+
+// Endpoint for handling debug metrics
+module.exports = (server) => {
+  const api = server.select('API')
+  const gauge = new client.Gauge({ name: 'number_of_peers', help: 'the_number_of_currently_connected_peers' })
+
+  setInterval(() => {
```

Comment:
Not sure about this approach. Guess we have two options, 1) (which is implemented here) is to once every X seconds, check the number of peers and save it or 2) when a request happens at `/debug/metrics/prometheus`, then check the number of peers and respond.

@diasdavid what you think?